Sources/Widgets/Representations/WidgetRepresentation/index.js

```diff
@@ -257,6 +257,7 @@ const DEFAULT_VALUES = {
     cameraPosition: [0, 0, 0],
     cameraDir: [1, 0, 0],
     isParallel: false,
+    rendererPixelDims: [1, 1],
```

Comment:
@floryst had to set it up to avoid crash. let me know if it is not a good default

Reply:
Seems good to me.